src/js/components/homepage/visualizations/treemap/TreeMapTooltip.jsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * TreeMapTooltip.jsx
+ * Created by Emily Gullo 04/28/2017
+ */
+
+import React from 'react';
+
+const propTypes = {
+    name: React.PropTypes.string,
+    value: React.PropTypes.string,
+    description: React.PropTypes.string,
+    height: React.PropTypes.number,
+    x: React.PropTypes.number,
+    y: React.PropTypes.number,
+    width: React.PropTypes.number,
+    showSub: React.PropTypes.bool,
+    percentage: React.PropTypes.string
+};
+
+export default class TreeMapTooltip extends React.Component {
+    componentDidMount() {
+        this.positionTooltip();
+    }
+    componentDidUpdate() {
+        this.positionTooltip();
+    }
+
+    positionTooltip() {
+        // we need to wait for the tooltip to render before we can full position it due to its
+        // dynamic width
+        const tooltipWidth = this.div.offsetWidth;
```

Comment:
I think there's some sizing and positioning issues with the tooltip on smaller screen sizes:
![screen shot 2017-05-12 at 11 53 54 am](https://cloud.githubusercontent.com/assets/3594363/26007610/44eabd48-370f-11e7-9927-82966a61208d.png)
![screen shot 2017-05-12 at 11 59 41 am](https://cloud.githubusercontent.com/assets/3594363/26007609/44e99594-370f-11e7-899e-be918ef03727.png)